draft-ietf-snac-simple.xml

```diff
@@ -395,13 +402,13 @@
 	    </ul>
 	  </section>
 	  <section anchor="state-deprecating">
-	    <name>Stub router deprecating its on-link prefix (STATE-DEPRECATING)</name>
+	    <name>Stub router deprecating its ULA Link Prefix (STATE-DEPRECATING)</name>
```

Comment:
Maybe we should leave this as it was—ULA Link prefix is ambiguous.

Reply:
Stub router deprecating the on-link prefix it is advertising

Reply:
Updated.